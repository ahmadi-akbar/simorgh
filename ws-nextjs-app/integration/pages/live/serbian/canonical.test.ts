/**
 * @service serbian
 * @pathname /serbian/live/media-23179005/lat
 */

import { runHeaderTests } from '../../../../../src/integration/common';

describe('Canonical', () => {
  describe(pageType, () => {
    runHeaderTests();
  });
});
