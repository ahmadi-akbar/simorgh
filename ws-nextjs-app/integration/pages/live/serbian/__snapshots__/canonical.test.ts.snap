// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canonical Live Header I can see the branding 1`] = `"BBC News, na srpskom"`;

exports[`Canonical Live Header Navigation link should match text and url 1`] = `
{
  "text": "Poƒçetna strana",
  "url": "/serbian/lat",
}
`;

exports[`Canonical Live Header Navigation link should match text and url 2`] = `
{
  "text": "Ukrajina",
  "url": "/serbian/lat/topics/c5wzvzzz5vrt",
}
`;

exports[`Canonical Live Header Navigation link should match text and url 3`] = `
{
  "text": "Srbija",
  "url": "/serbian/lat/topics/cr50vdy9q6wt",
}
`;

exports[`Canonical Live Header Navigation link should match text and url 4`] = `
{
  "text": "Balkan",
  "url": "/serbian/lat/topics/c06g87137jgt",
}
`;

exports[`Canonical Live Header Navigation link should match text and url 5`] = `
{
  "text": "Svet",
  "url": "/serbian/lat/topics/c2lej05e1eqt",
}
`;

exports[`Canonical Live Header Navigation link should match text and url 6`] = `
{
  "text": "Video",
  "url": "/serbian/lat/topics/c44vyp5g049t",
}
`;

exports[`Canonical Live Header Navigation link should match text and url 7`] = `
{
  "text": "Najpopularnije",
  "url": "/serbian/lat/popular/read",
}
`;
